lesson_08/script.js

```diff
@@ -0,0 +1,45 @@
+"use strict";
+const ul = document.getElementById("js-list");
+// ローディング作る
+const addLoading = () => {
+  const newElement = document.createElement("div");
+  const gif = document.createElement("img");
+  newElement.id = "js-loading";
+  gif.src = "loading-circle.gif";
+  ul.appendChild(newElement).appendChild(gif);
+};
+// ローディング消す
+const removeLoading = () => document.getElementById("js-loading").remove();
+// リスト
+const getData = new Promise((resolve, reject) => {
+  addLoading();
+  const data = [
+    { to: "bookmark.html", img: "1.png", alt: "画像1", text: "ブックマーク" },
+    { to: "message.html", img: "2.png", alt: "画像2", text: "メッセージ" }
+  ];
+  setTimeout(() => {
+    resolve();
+    reject("error");
+  }, 3000);
+});
+
+getData
+  .then((value) => {
+    const ul = document.getElementById("js-list");
+    const fragment = document.createDocumentFragment();
+    for (let i = 0; i < value.length; i++) {
+      const li = document.createElement("li");
+      const a = document.createElement("a");
+      const img = document.createElement("img");
+      a.href = value[i].to;
+      a.textContent = value[i].text;
+      img.src = value[i].img;
+      img.alt = value[i].alt;
+      fragment.appendChild(li).appendChild(a).appendChild(img);
+    }
+    removeLoading();
+    ul.appendChild(fragment);
+  })
+  .catch((error) => {
+     console.error();
```

Comment:
![image](https://user-images.githubusercontent.com/81888887/162551433-593af33e-3557-43be-af3b-af2936ebcd22.jpeg)
Since the argument is not set, the error content is not displayed. Think about it a little more.